### PR TITLE
merge boolean fix

### PR DESCRIPTION
**Pull request by msenmurugan**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/306#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/util/crd_defaults.go [hunk 1]
```diff
@@ -652,7 +652,7 @@ func GetAlertDefault() *alertv1.AlertSpec {
 		Port:                 &port,
 		EncryptionPassword:   "",
 		EncryptionGlobalSalt: "",
-		PersistentStorage:    true,
+		PersistentStorage:    false,
```
Comment (jdartigalongue) [inline pkg/util/crd_defaults.go]: Synopsysctl uses this template when generating Alert custom resources. The persistent storage for that one should remain true

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/306#issuecomment-490312893" title="LGTM">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/306#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue,msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by msenmurugan** [comment]
/cherrypick-release-2019.4.x

**Comment by operator-r2d2** [comment]
@rrati: cannot checkout 2019.4.x: error checking out 2019.4.x: exit status 1. output: error: pathspec '2019.4.x' did not match any file(s) known to git.


<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/306#issuecomment-490313142):

>/cherrypick 2019.4.x


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: new pull request created: #307

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/306#issuecomment-490313177):

>/cherrypick release-2019.4.x
>
>


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**